src/main/kotlin/racingcar/domain/RandomStrategy.kt

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain
+
+import kotlin.random.Random
+
+class RandomStrategy : MovingStrategy {
```

Comment:
```suggestion
object RandomStrategy : MovingStrategy {
```
클래스는 하나 이상을 캡슐화하는 것이 좋아요.
캡슐화할 것이 없다면, object로 만들어주시면 좋아요. (이하 관련 내용 동일)